src/view/dataElements/xdmObject/helpers/schemaSelection/loadDefaultSandbox.js

```diff
@@ -27,15 +28,21 @@ const loadDefaultSandbox = async ({
       imsAccess
     }));
   } catch (e) {
-    console.error(e);
-    reportAsyncError(new Error(`Failed to load sandboxes. ${e.message}`));
+    reportAsyncError(
+      new UserReportableError("Failed to load sandboxes.", {
+        originatingError: e
+      })
+    );
     return undefined;
   }
 
   if (!sandboxes.length) {
     reportAsyncError(
-      new Error(
-        "You do not have access to any sandboxes. Please contact your administrator to be assigned appropriate rights."
+      new UserReportableError(
+        "You do not have access to any sandboxes. Please contact your administrator to be assigned appropriate rights.",
+        {
+          additionalInfoUrl: "https://adobe.ly/3gHkqLF"
```

Comment:
@jfkhoury was thinking a link would be nice for this error, so this is how it's done.